platform/mellanox/mlnx-platform-api/sonic_platform/vpd_parser.py

```diff
@@ -96,4 +97,21 @@ def get_entry_value(self, key):
             return 'N/A'
         return self.vpd_data.get(key, 'N/A')
 
+    def get_dpu_data(self, key=None):
```

Comment:
Will it be better to add a DpuVpdParser(VpdParser) class?

Reply:
Created a new `DpuVpdParser` class